src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -189,43 +193,48 @@ Dictionary
                           Other &nbsp;&nbsp;
                         <form className="form-inline search-bar">
                           <i className="fas fa-search" />
-                          { otherSelected && <input
+                          {otherSelected && <input
                             {
-                            ...getInputProps()
-                            }
+                              ...getInputProps()
+                              }
                             className="form-control search"
                             id="sourceSearch"
                             placeholder="Search"
                             aria-label="Search"
                           />
-                          }
+                            }
+                          {isFetching
+                              && <div className="ml-auto text-right">
+                                <Loader smaller />
+                              </div>
+                            }
                         </form>
                       </div>
                       <div className="search-ul">
                         {isOpen ? (
                           <ul {...getMenuProps()} className="search-ul">
                             {
-                            conceptSources.filter(item => !inputValue.trim()
```

Comment:
Is this just a formatting change?

Reply:
Yes, It was because I added the div for the loader component